packages/app/src/cli/commands/app/generate/extension.ts

```diff
@@ -74,26 +84,59 @@ export default class AppScaffoldExtension extends Command {
     const {flags} = await this.parse(AppScaffoldExtension)
 
     await metadata.addPublic(() => ({
-      cmd_scaffold_required_auth: false,
+      cmd_scaffold_required_auth: true,
       cmd_scaffold_template_custom: flags['clone-url'] !== undefined,
       cmd_scaffold_type_owner: '@shopify/app',
```

Comment:
This is intended to be
> For now always @shopify / app -- all types registered by one plugin. But in the future custom plugins might register pre-release types

So I think I'd expect this to change -- if the scaffolded extension was registered by a different plug-in.

Reply:
Got it, we are still not loading extensions from plugins, but i'll take this into account when we do